05_stringer/selva/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIPAddString(t *testing.T) {
```

Comment:
This should be called `TestMainOuput`.
Write another function called `TestIPAddString` using [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests) where you only test the string output. e.g.:
```go
var testData = []struct {
	in  IPAddr
	out string
}{
	{IPAddr{127, 0, 0, 1}, "127.0.0.1"},
	{IPAddr{1}, "1.0.0.0"},
}

func TestIPAddString(t *testing.T) {
	r := require.New(t)
	for _, tt := range testData {
		r.Equalf(tt.out, tt.in.String(), "Convert IP address to dotted quad fails")
	}
}
```